recipes/ace/build.sh

```diff
@@ -0,0 +1,38 @@
+#!/bin/sh
+
+# Installation following the instructions in 
+# https://htmlpreview.github.io/?https://github.com/DOCGroup/ACE_TAO/blob/master/ACE/ACE-INSTALL.html#unix
+export ACE_ROOT=${SRC_DIR}
+export ACE_SOURCE_PATH=${ACE_ROOT}/ace
+export WORKSPACE=${ACE_ROOT}/ace/ace
+export INSTALL_PREFIX=$PREFIX
+export PERL_PATH=$CONDA_PREFIX/bin/perl
+
+# Configure step
+cd ${ACE_ROOT}
+perl ${ACE_ROOT}/bin/mwc.pl -type gnuace -features "uses_wchar=1,zlib=0,ssl=0,openssl11=0,trio=0,xt=0,fl=0,fox=0,tk=0,qt=0,rapi=0,stlport=0,rwho=0" ${WORKSPACE}.mwc
+
+if [ -z "$MACOSX_DEPLOYMENT_TARGET" ]
```

Comment:
Is this the correct check?

Reply:
I checked for the environment variables that are only defined on macOS, I do not know if some conda expert (@wolfv @Tobias-Fischer ?) can suggest a better check to have conditional code only on macOS.

Reply:
there is the `$target_platform` env var which would be `osx-64`, or `osx-arm64`, but many people also chekc using `uname`. 

A real life-saver is the org-wide search on github: https://github.com/search?q=org%3Aconda-forge+target_platform&type=code

MACOSX_DEPLOYMENT_TARGET might also soon be deprecated (or mabye already is :/)